htdocs/ticket/card.php

```diff
@@ -738,7 +738,21 @@
 
 if ($action == 'create' || $action == 'presend') {
 	$formticket = new FormTicket($db);
+	if (!GETPOSTISSET('token')) {
```

Comment:
In other part of code, we used a different solution:
The parameter mode=init was added on the link with action=presend
And we reset dir and session when mode=init. For eample in htdocs/admin/mails.php
```
		// Init list of files
		if (GETPOST("mode", "aZ09") == 'init') {
			$formmail->clear_attached_files();
		}

```
When mode is not init, we untouch dir

Reply:
I have taken your comment into account to use mode=init as suggested. This results in a modification of modTiket and an SQL update. Let me know if this works for you.

Reply:
@eldy